controllers/stories.js

```diff
@@ -0,0 +1,80 @@
+const stories = require('../models/stories')
+/**
+ * Creates new story
+ *
+ * @param req {Object} - Express request object
+ * @param req.body {Object} - Story object
+ * @param res {Object} - Express response object
+ */
+const addNewStory = async (req, res) => {
+  try {
+    const body = {
+      ...req.body
+    }
+    const story = await stories.updateStory(body)
+
+    if (story) {
+      return res.json({
+        message: 'Story created successfully!',
+        story: story.storyDetails,
+        id: story.storyId
+      })
+    } else {
+      logger.error('Error while creating new story: Incorrect username passed')
+      return res.boom.badRequest('Unable to add story')
+    }
```

Comment:
I believe this code block will be unused if the promise on the line no 14 fails
The execution will jump in the `catch`.

Reply:
Yes, but I have added this code block in case the username passed for frontend engineer, backend engineer or feature owner is invalid then the promise returns false, and we send a 400 response